src/com/textFinder/finder/FileSearchThreadRunner.java

```diff
@@ -0,0 +1,28 @@
+package com.textFinder.finder;
+
+import java.io.File;
+import java.util.List;
+
+public class FileSearchThreadRunner implements Runnable {
```

Comment:
`FileSearchThreadRunner` <> `FileSearchThreadCallable implements Callable<Boolean>`

And from the call method, return if this file contains the search text or not